Sources/tuist/tuist.docc/Articles/Tuist Cloud/Tuist Cloud - Command Line Interface.md

```diff
@@ -0,0 +1,342 @@
+# Command Line Interface
```

Comment:
I'm not a big fan of duplicating source of truth, I would just keep the very basic or the tutorial here, and leave the specifics to the `tuist cloud --help`, `tuist cloud init --help`, etc.
Otherwise, this is very likely to become obsolete/wrong soon

Reply:
I agree it's not great to duplicate source of truth but I think it's useful to have the command line interface fully in the docs. Ideally, we'd be able to partly generate it.

Reply:
I think it's fine keeping the reference to the commands and the examples, but I wouldn't go as deep as documenting the parameters

Reply:
Good point! I'll drop the documentation for now, and in a different effort go through all the commands making sure that the documentation is relevant and that it mentions whether they are or not interactive and the authorization model.

Reply:
I pushed a commit dropping the changes. I also created [a PR](https://github.com/tuist/docs/pull/32) in the current documentation doing the same there.